sql/builtin_functions.h

```diff
@@ -152,10 +153,10 @@ struct RegisterBuiltin {
 
                     bool constantArgs = true;
                     for (auto& arg : args) {
-                        constantArgs = constantArgs && arg.metadata.isConstant;
+                        constantArgs = constantArgs && arg.info->isConst();
                     }
 
-                    result.resultMetadata.isConstant = constantArgs && determinism == DETERMINISTIC;
+                    result.resultInfo = result.resultInfo->getConst(constantArgs && determinism == DETERMINISTIC);
```

Comment:
If we find one we can see if its worth tackling it as an optimization. Being falsely non-const has no impact other than performance.